scipy/cluster/hierarchy.py

```diff
@@ -3004,13 +3004,70 @@ def set_link_color_palette(palette):
     _link_line_colors = palette
 
 
+def reorder_leaves(Z, leaves_order):
```

Comment:
I assume this is meant to be a private function right? Could you prepend an underscore to the name?

Reply:
Will do in the next commit.